pkg/apis/core/validation/validation_test.go

```diff
@@ -10230,12 +10230,18 @@ func TestValidatePodSpec(t *testing.T) {
 				SupplementalGroupsPolicy: &goodSupplementalGroupsPolicy,
 			}),
 		),
+		"populate PodLevelResources with valid OS": podtest.MakePod("",
```

Comment:
Is there a test case with OS field == nil?

Reply:
Added.